pkgs/development/compilers/llvm/5/llvm-outputs.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/tools/llvm-config/llvm-config.cpp b/tools/llvm-config/llvm-config.cpp
+index 94d426b..37f7794 100644
+--- a/tools/llvm-config/llvm-config.cpp
++++ b/tools/llvm-config/llvm-config.cpp
+@@ -333,6 +333,21 @@ int main(int argc, char **argv) {
+     ActiveIncludeOption = "-I" + ActiveIncludeDir;
+   }
+ 
```

Comment:
Btw is the output of `llvm-config` sane in this release?

Reply:
I gc'd my local build of this, whoops, so after that finishes I'll poke at it and check it looks reasonable when compared against the output given by LLVM 4's llvm-config.

If there's anything in particular you'd like to confirm works, LMK.

Reply:
This pointed to static libraries in older unpatched versions:

```
$ llvm-config  --libfiles
/nix/store/7sa6x0qzqxnxlf9jy2vih66yzy12sja4-llvm-4.0.1-lib/lib/libLLVM-4.0.so
```